sdk/tests/conformance/extensions/ext-sRGB.html

```diff
@@ -311,7 +311,7 @@
   var formatString;
   var validFormat;
   switch (format) {
-    case ext.SRGB_EXT: formatString = "sRGB"; validFormat = false; break;
+    case ext.SRGB_EXT: formatString = "sRGB"; validFormat = true; break;
```

Comment:
Thank you, validFormat should be false, 'SRGB_EXT' is definitely a non color-renderable format.